cuda_bindings/cuda/bindings/utils/_get_handle.pyx.in

```diff
@@ -1,234 +0,0 @@
-# SPDX-FileCopyrightText: Copyright (c) 2021-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
-# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
-
-# This code was automatically generated with version 13.0.0. Do not modify it directly.
-
-from libc.stdint cimport uintptr_t
-cimport cython
-
-from cuda.bindings cimport driver, runtime, cydriver, cyruntime
-
-
-cdef dict _handle_getters = None
-
-@cython.embedsignature(True)
-def get_cuda_native_handle(obj) -> int:
```

Comment:
We should double check that removing this method doesn't break the ABI and that if someone built against a prior version that there isn't an issue at runtime due to the removal of this module / function.

Reply:
This function is `def` not `cdef`, so there is no ABI issue. It's only accessible from within Python.